source/version_helper.js

```diff
@@ -24,6 +24,17 @@ module.exports = {
   GetCockroachDBVersionFromEnvConfig: function() {
     const crdbVersion = process.env['CRDB_VERSION'] 
     return semver.coerce(crdbVersion)
+  },
+  GetVersionSeries: function(versionStr) {
+    // Get the version series from a version string.
+    // E.g. "6.0.1" is of series "6.0".
+    const regExp=/^(\d+\.\d+)(\.|$)/mg
+    let match = regExp.exec(versionStr);
+    if (match === null || match.length < 3) {
+      console.log(`the version string of Sequelize is not of xx.xx format, where x is a digit`)
```

Comment:
Don't think we should have this console.log here right?

Reply:
@RichardJCai  Just removed this console.log. Thanks!